### ISSUE TITLE
Update Vault Values without New Commit in Argo

### ISSUE THREAD

**Issue by jhoover4** [IBM/argocd-vault-plugin#179]
**Question**

Hey guys - this might be more of a question or feature request?

Something we're running into a lot is that I believe once Argo pins a commit, it takes the values in Vault at the time of that commit and applies them through the plugin, and caches whatever was applied.

That's fine, but sometimes we need to change values in Vault and refresh the secret (in the same commit). I know that feels kinda hacky and might go against GitOps, but I'm wondering if there's a way to accomplish that with the plugin?

So in other words is there a way to change values in a Vault object and have Argo reflect those new values without deploying a new commit?

**Comment by werne2j** [comment]
Hard Refresh in Argo CD will solve this for you https://ibm.github.io/argocd-vault-plugin/usage/#refreshing-values-from-secrets-managers

**Comment by jhoover4** [comment]
Wow okay thanks! Hadn't thought to check the new docs, I forgot about that.
